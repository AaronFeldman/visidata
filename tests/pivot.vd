sheet	col	row	keystrokes	input	comment
	sheet	0	o	sample_data/sample.tsv	open input in VisiData
sample	Rep	0	!		pin current column to the left as a key column
sample	Units	0	#		set type of current column to int
sample	Units	0	+	sum	add aggregator to the current column
sample	Item	0	W		open Pivot Table
