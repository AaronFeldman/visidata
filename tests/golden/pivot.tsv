Rep	Total_Units_sum	Units_sum_Pencil	Units_sum_Binder	Units_sum_Pen	Units_sum_Desk	Units_sum_Pen Set	Total_count
Jones	396	130	124	64	0	78	8
Kivell	193	0	50	0	5	138	4
Jardine	281	126	105	0	0	50	5
Gill	213	60	126	27	0	0	5
Sorvino	142	56	7	76	3	0	4
Andrews	183	155	28	0	0	0	4
Thompson	89	32	57	0	0	0	2
Morgan	173	90	28	0	0	55	3
Howard	125	0	29	96	0	0	2
Parent	170	0	81	15	0	74	3
Smith	156	67	87	0	2	0	3
